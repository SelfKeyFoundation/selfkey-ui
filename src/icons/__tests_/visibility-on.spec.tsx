import * as React from 'react';
import { create } from 'react-test-renderer';
import { VisibilityOnIcon } from '../visibility-on';

describe('Visibility on Icon', () => {
  it('should show Visibility on Icon', () => {
    const component = create(
      <VisibilityOnIcon/>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});





