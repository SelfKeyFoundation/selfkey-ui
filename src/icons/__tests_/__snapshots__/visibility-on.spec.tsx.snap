// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Visibility on Icon should show Visibility on Icon 1`] = `
<svg
  aria-hidden="true"
  className="MuiSvgIcon-root"
  focusable="false"
  style={
    Object {
      "height": "24px",
      "width": "24px",
    }
  }
  viewBox="0 0 24 24"
>
  <g>
    <path
      d="M0 0h24v24H0z"
      fill="none"
    />
    <path
      d="M12 4.5C7 4.5 2.73 7.61 1 12c1.73 4.39 6 7.5 11 7.5s9.27-3.11 11-7.5c-1.73-4.39-6-7.5-11-7.5zM12 17c-2.76 0-5-2.24-5-5s2.24-5 5-5 5 2.24 5 5-2.24 5-5 5zm0-8c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"
    />
  </g>
</svg>
`;
